src/redux/action.types.js

```diff
@@ -0,0 +1,3 @@
+export const DATA_STREAM_REQUEST = 'DATA_STREAM_REQUEST';
```

Comment:
because the name is ..._REQUEST i thought it's an API request. how do you get the data streams? or they are hardcoded?